config/includes.chroot/etc/initramfs-tools/scripts/init-premount/loop_mount

```diff
@@ -21,60 +19,79 @@ if [ "x${LOOPROOT}" != "x" ]; then
 	modprobe loop
 	mkdir -p ${lprootp}
 
-	log_begin_msg "mounting ${LOOPROOT}"
-	mount -t ntfs-3g ${LOOPROOT} ${lprootp} || {
+	_log_msg "Mounting ${LOOPROOT} to ${lprootp} ...\n"
+	/usr/bin/mount ${LOOPROOT} ${lprootp} || {
 		rc=$?
 		log_failure_msg "mounting failed ${rc}"
 		exit ${rc}
 	}
-	log_end_msg
+	_log_msg "... mounting done.\n"
+
+	case ${LIOXWIPE} in
+	1)
+		_log_msg "Deleting liox-*.raw from ${LOOPROOT}\n"
+		rm -v -f ${LOOPROOT}/liox-*.raw
```

Comment:
busybox version of `rm` does not output anything when `-v` is passed. You`ll need to get GNU rm for this too.